### PR TITLE
Allow Flutter shell to build for a Mac Fuchsia host.

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: sky/engine/platform/BUILD.gn [hunk 1]
```diff
@@ -97,7 +97,6 @@ source_set("platform") {
     "fonts/harfbuzz/HarfBuzzFaceSkia.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.cpp",
     "fonts/harfbuzz/HarfBuzzShaper.h",
-    "fonts/linux/FontPlatformDataLinux.cpp",
```
Comment (pylaligand) [inline sky/engine/platform/BUILD.gn]: Btw not sure how this worked before as the combo of this + FontPlatformDataApple generates duplicate symbols on Mac.

Comment (abarth) [inline sky/engine/platform/BUILD.gn]: Maybe we had a fancy source filter somehow?

Comment (pylaligand) [inline sky/engine/platform/BUILD.gn]: Ah yeah, that's probably it. //buildroot/build/BUILDCONFIG.gn uses set_sources_assignment_filter.

**Pull request merged**